src/Muqsit/SendPlayerFaceTask.php

```diff
@@ -102,17 +102,35 @@ public function onRun()
     {
         $symbol = hex2bin(self::HEX_SYMBOL);
         $strArray = [];
-        $skin = substr($this->skindata, ($pos = (64 * 8 * 4)) - 4, $pos);
-	for($y = 0; $y < 8; ++$y){
-		for($x = 1; $x < 9; ++$x){
+
+        switch(strlen($this->skindata)){
+            case 8192:
+            case 16384:
+                $maxX = $maxY = 8;
+
+                $width = 64;
+                $layer2Offset = 32;
```

Comment:
I should've called this uv I suppose.

Reply:
yeah, uv sounds better

Reply:
Btw, line 106 to 122 fomatting (replace spaces with tabs)